core/src/main/scala/com/databricks/labs/remorph/utils/Strings.scala

```diff
@@ -60,4 +62,25 @@ object Strings {
       seq.mkString(start, sep, end)
     }
   }
+
+  /**
+   * Formats the text of a given `ParserRuleContext` by concatenating the text of all its children nodes,
+   * separated by spaces.
+   *
+   * @param ctx the `ParserRuleContext` to format.
+   * @return a `String` representing the concatenated text of all children nodes, separated by spaces.
+   *         If the context has no children, returns an empty string.
+   */
+  def formatContextString(ctx: ParserRuleContext): String = {
```

Comment:
this method belongs better to com.databricks.labs.remorph.parsers.{IncompleteParser or ParserCommon}. I also do think that we should return not a String, but rather `UnresolvedCommand`.

Reply:
ok will reorganise.